python/lib/dependabot/python/update_checker/pip_compile_version_resolver.rb

```diff
@@ -251,7 +251,13 @@ def pip_compile_options(filename)
             options << "--output-file=#{requirements_file.name}"
           end
 
-          options.join(" ")
+          command = "pyenv exec pip-compile"
+          if (requirements_file = compiled_file_for_filename(filename)) &&
+             requirements_file.content.include?("autogenerated by uv")
```

Comment:
Sorry we meant to have both be the same. It would be worth extracting but I wouldn't know how to do that quickly, if you have a suggestion of where and how that would be done I would appreciate it very much.
```suggestion
             requirements_file.content.include?("uv pip compile")
```